pyproject.toml

```diff
@@ -38,7 +38,7 @@ jsonschema = ">=4.10"
 cryptography = ">42.0.0"
 pydantic ="^1.10.7"
 pyyaml = "^6.0.1"
-lightkube = "^0.15.0"
+lightkube = "0.15.0"
```

Comment:
**question**: Did you experience breaking changes in the `.2` and `.3` patch releases? The changelogs are quite sparse, when they exist.
Also, why do we need `lightkube` in the lint group?

Reply:
I was having huge issues in the typing inference of it, actually opened an issue for it - https://www.github.com/gtsystem/lightkube/issues/63
It 'seems' to be resolved, but we can look at it in another PR.